SF iOS/SF iOS/Feed/SwiftUI/Data.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  Data.swift
+//  Coffup
+//
+//  Created by Roderic Campbell on 6/8/19.
+//
+
+import UIKit
+import SwiftUI
+import CoreLocation
+import Combine
+
+struct WrappedEvent /*, BindableObject*/ {
+//    var didChange = (NotificationCenter.default.publisher(for: "eventChange") as AnyObject).receive(on: RunLoop.main)
+//    var event: Event
+    let name: String
+    let eventID: String
+    let startDate: String
+    let venue: Venue
+    var locationCoordinate: CLLocationCoordinate2D {
+        CLLocationCoordinate2D(
+            latitude: venue.location.latitude,
+            longitude: venue.location.longitude)
+    }
+    
+    enum CodingKeys: String, CodingKey {
+        case eventID = "id"
+        case name = "name"
+        case venue = "venue"
+        case startDate = "start_at"
+    }
+}
+
+struct Venue: Hashable, Codable {
+    let name: String
+    let location: Location
+    enum CodingKeys: String, CodingKey {
+        case name = "name"
+        case location = "location"
+    }
+}
+
+struct Location: Hashable, Codable {
+    let address: String
+    let latitude: Double
+    let longitude: Double
+    
+    enum CodingKeys: String, CodingKey {
+        case address = "formatted_address"
+        case latitude = "latitude"
+        case longitude = "longitude"
+    }
+}
+
+extension WrappedEvent: Decodable {
+}
+
+
+let eventData: [WrappedEvent] = load("eventData.json")
+
+func load<T: Decodable>(_ filename: String, as type: T.Type = T.self) -> T {
+    let data: Data
+    print("this is good")
```

Comment:
smh

Reply:
extension WrappedEvent: Decodable {}?

Reply:
I was SMH at the ` print("this is good")` because I should have taken that out :(